sycl/include/CL/__spirv/spirv_vars.hpp

```diff
@@ -15,7 +15,7 @@
 
 #define __SPIRV_VAR_QUALIFIERS extern "C" const
 
-#if defined(__SYCL_NVPTX__)
+#if defined(__NVPTX__) && defined(__SYCL_DEVICE_ONLY__)
```

Comment:
```suggestion
#if defined(__NVPTX__)
```
This code is already under `#ifdef __SYCL_DEVICE_ONLY__`. See line 14.

Reply:
Thanks, done in https://github.com/intel/llvm/pull/3977/commits/a9df5084c8d2ad2a9b5f31e76c34221d6955a9aa.